blessedDeps.gradle

```diff
@@ -68,7 +68,8 @@ rootProject.ext.deps = [
     androidArchCoreTesting : "android.arch.core:core-testing:$ANDROID_ARCH_TESTING",
     androidTestRunner      : "com.android.support.test:runner:$ANDROID_TEST_RUNNER",
     androidAnnotations     : "androidx.annotation:annotation:$ANDROIDX_ANNOTATION",
-    androidTestCore        : "androidx.test:core:1.2.0",
+    androidTestCore        : "androidx.test:core:1.3.0",
```

Comment:
I needed to update this to run the activity scenario tests. Otherwise the tests would crash.